dist/index.html

```diff
@@ -18,24 +18,15 @@ <h1>Leaderboard</h1>
           <h2>Recent scores</h2>
           <button>Refresh</button>
```

Comment:
- [ ] Good job so far but this button is not responsive because you have not given it any class but in your `js` file you are referencing the class `.refBtn`. Kindly address this issue and make sure the button works and it updates the lists of scores 👍